lib/src/test/scala/code/winitzki/benchmark/DiningPhilosophersSpec.scala

```diff
@@ -23,11 +23,11 @@ class DiningPhilosophersSpec extends FlatSpec with Matchers with TimeLimitedTest
 
     val tp = new FixedPool(8)
 
-    val h1 = new M[Int]("Aristotle is hungry")
-    val h2 = new M[Int]("Kant is hungry")
-    val h3 = new M[Int]("Marx is hungry")
-    val h4 = new M[Int]("Russell is hungry")
-    val h5 = new M[Int]("Spinoza is hungry")
+    val h1 = new M[Int]("Aristotle is eating")
```

Comment:
Actually, I think the names are fine. Molecule h1 is present only while the philosopher 1 is hungry. When the philosopher is eating, h1 has been consumed by a reaction that prints "philosopher is eating". This is what confused me when I was writing the molecule names.